src/odemis/driver/test/smaract_test.py

```diff
@@ -446,7 +446,6 @@ def test_reference_cancel(self):
             # procedure is started, so this doesn't work.
             # TODO: extend the simulator to report axis is referenced only at the end of referencing.
             f = self.dev.reference()
-            time.sleep(0.1)
```

Comment:
Leave this line here. It has nothing to do with the script (and the line is actually needed).